playbooks/etc_hosts/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: "Build hosts file"
+  lineinfile: dest=/etc/hosts regexp='.*{{ item.replace('.', '-') }}$' line="{{ hostvars[item].ansible_ssh_host }} {{ item.replace('.', '-') }}.{{ etc_hosts_domain }} {{ item.replace('.', '-') }}" state=present
```

Comment:
@stbenjam I had to make a small change here to replace any dots with dashes. The reason being for our version boxes the name includes a dot and we correct for that in the hostname declaration -- https://github.com/Katello/katello-deploy/blob/master/Vagrantfile#L48

The longer term solution may be to declare our version boxes with dashes from the outset (e.g. centos7-katello-3-0) for consistency, however, that doesn't prevent custom boxes from using dots unless we add and throw validation errors.